data/presets/leisure/parklet.json

```diff
@@ -0,0 +1,28 @@
+{
+    "icon": "maki-picnic-site",
+    "fields": [
+        "name",
+        "parklet/seatings",
+        "parklet/greenery",
+        "material",
+        "colour",
+        "seasonal"
+    ],
+    "moreFields": [
+        "level",
+        "manufacturer",
+        "operator",
+        "wheelchair"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "leisure": "parklet"
+    },
+    "terms": [
+        "parklet"
```

Comment:
```suggestion
        "bench",
        "outdoor seating",
        "parking"
```

Reply:
I don't really understand, why remove "parklet" and add "parking"?